ontology/docs/faq.md

```diff
@@ -0,0 +1,115 @@
+# Frequently Asked Questions
+
+This document is intended to provide answers to frequently asked questions regarding the application of the Digital Buildings ontology to existing and novel (i.e. previously unmodeled) devices.
+
+[TOC]
+
+## What is a building model?
+**A building model is a file that maps concrete real-world devices to the ontology.** 
+
+The model takes the form of a building configuration file (see [here](building_config.md)). It contains all the relevant information for the building: what devices exist, what types they apply, which devices connect to which other devices, what devices serve which zone, etc.
```

Comment:
might need ./building_config.md

Reply:
i dont think it does... we dont have those in the links in other ontology docs in the same folder.